tests/factories/layer_data.py

```diff
@@ -0,0 +1,28 @@
+import factory
+
+from snowexsql.tables import LayerData
+from .base_factory import BaseFactory
+from .doi import DOIFactory
+from .instrument import InstrumentFactory
+from .measurement_type import MeasurementTypeFactory
+from .site import SiteFactory
+
+
+class LayerDataFactory(BaseFactory):
+    class Meta:
+        model = LayerData
+
+    depth = 100.0
+    pit_id = 'Pit 123'
+    bottom_depth = 90.0
+    comments = 'Layer comment'
+    value = '40'
+    flags = 'Sample'
+    sample_a = '42'
```

Comment:
@micah-prime - Can you remind me, why the `value` `and sample_(a|b|c)` columns are strings and not int column types?

Reply:
We're storing the Grain Type, Hand Hardness, and Manual Wetness data in the value column as well. I'm assuming the a, b, and c samples are for consistency with that data type